RELEASING.md

```diff
@@ -1,6 +1,6 @@
 # Releasing EAS CLI
 
-1. Run `yarn release` in the repository root folder. The next version is chosen automatically based on the changelog entries. If you want to use different version, pass the version string as a positional argument to the command, e.g. `yarn release 1.2.3`.
+1. Invoke the GitHub workflow "Manually trigger a release" (`trigger-release.yml`). The next version is chosen automatically based on the changelog entries. If you want to use different version, pass the version string as an input to the workflow.
```

Comment:
I would find it helpful to hyperlink to the workflow page, and also provide an example of the version string, specifically to differentiate between whether it should be a plain semver version like `1.2.3` or have a v prefix like `v1.2.3` or something else.

Reply:
Done!